static/css/restyle.less

```diff
@@ -580,6 +580,15 @@ button.search-button {
   margin-top: 0;
 }
 
+.toplist li a {
+  line-height: 1.4em;
```

Comment:
Could this be better without a unit? [1]

[1] https://developer.mozilla.org/en/docs/Web/CSS/line-height#Prefer_unitless_numbers_for_line-height_values